ui/src/app/api/auth/[...nextauth]/route.ts

```diff
@@ -0,0 +1,37 @@
+import NextAuth from "next-auth/next";
+import GitHubProvider from "next-auth/providers/github"
+import CredentialsProvider from "next-auth/providers/credentials"
+
+export const authOptions = {
+    providers: [
+        GitHubProvider({
+            clientId: process.env.GITHUB_ID ?? "",
+            clientSecret: process.env.GITHUB_SECRET ?? ""
+        }),
+        CredentialsProvider({
+            name: "Credentials",
+            credentials: {
+                email: {label: "Email", type: "text", placeholder: "email"},
+                password: {label: "Password", type: "password"}
+            },
+            async authorize(credentials, req) {
+                const res = await fetch('http://localhost:3333/auth/login', {
```

Comment:
I would be nice to have server url as variable in .env and concrete endpoints in objects to use it like this fetch(`${ENV_VARIABLE}/${endpoints.auth.login}`)

Reply:
Such usage is less error prone